app/views/main/_header.html.erb

```diff
@@ -82,12 +82,10 @@
                   </div>
                 </div>
                 <div>
-                  <%= f.label "medium", class: "block text-sm font-medium leading-6 text-gray-900" %>
+                  <%= f.label "medium", class: "block text-sm font-medium leading-6 text-gray-900 mb-1" %>
                   <div id="medium-item-container" class="flex flex-col gap-1">
-                    <span class="text-slate-500 text-sm">Filter medium options</span>
-                    <input type="text" class="w-48 small-field mb-2" id="medium-serach" />
                     <% @medium_options.each do | medium | %>
-                      <div data-text="<%= medium.id %>" class="medium-item w-full <%= "hidden opacity-0" unless medium.show_by_default %> transition-opacity duration-500 delay-75 ">
+                      <div data-text="<%= medium.id %>" class="medium-item w-full transition-opacity duration-500 delay-75 ">
```

Comment:
Will medium have an `id` after these changes?

Reply:
Thanks. I checked that. It is basically the `medium` name that is being added here for the js code to show/hide items